source/layouts/article_layout.haml

```diff
@@ -0,0 +1,42 @@
+:ruby
+  site_content_server = data.site.content_server_url
+  -# site_asset_server = data.site.asset_server_url
+  article_title = current_article.data.title
+  article_author = current_article.data.author
+  article_date = current_article.date
+  article_image = current_article.data.image.to_s
+  article_image_lg = current_article.data.image_lg.to_s
+  article_caption = current_article.data.caption.to_s
+
+= wrap_layout :layout do
+  - content_for :meta_desc do
+    = tag "meta", name: "description", content: article_title
+
+  %article
+    %section.container
```

Comment:
We need to add some margin here so that text doesn't run all the way to the edge of the screen on mobile. I do this using the Neat grid, which adds gutters automatically. There I'm using Neat 2.0, and here we're still on Neat 1.x. But I think we can accomplish the same thing with gutters. So don't just throw a margin in there, check the grid offerings and try to use that. We actually do it that way elsewhere in the ETO site I believe. (see second screenshot)

![screenshot 2017-09-21 11 37 26](https://user-images.githubusercontent.com/615841/30686588-6e69a43a-9ec1-11e7-91e5-a0ca6127092f.png)

![screenshot 2017-09-21 11 42 57](https://user-images.githubusercontent.com/615841/30686817-0d17eaa6-9ec2-11e7-86f0-0f0dfe8e3805.png)